setup.py

```diff
@@ -56,7 +56,9 @@ def get_version():
         'pillow>=8.3',
         'pillow-jpls>=1.0',
         'pylibjpeg>=1.4',
-        'pylibjpeg-libjpeg>=1.3',
         'pylibjpeg-openjpeg>=1.2',
```

Comment:
What do you think about making them optional as well (I have repeatedly had trouble installing them)? The `libjpeg` optional dependency flag would then install `pylibjpeg`, `pylibjpeg-libjpeg`, and `pylibjpeg-openjpeg`.

We should have basic support for JPEG-2000 through the Pillow library.

We may even make `pillow-jpls` optional, too. We would then provide basic support for baseline JPEG and JPEG-2000 through Pillow. Support for additional transfer syntaxes would require installing optional dependencies.

Reply:
I agree that this probably makes sense